docs/source/create_channel/create_channel_genesis.md

```diff
@@ -0,0 +1,283 @@
+
+The `configtx.yaml` used by the test network contains two channel profiles. The first profile, `TwoOrgsOrdererGenesis:`, is used to create the genesis block of the system channel. The genesis block of the system channel is then used by the ordering node to create the system channel before any other channels are created:
+```yaml
+TwoOrgsOrdererGenesis:
+    <<: *ChannelDefaults
+    Orderer:
+        <<: *OrdererDefaults
+        Organizations:
+            - *OrdererOrg
+        Capabilities:
+            <<: *OrdererCapabilities
+    Consortiums:
+        SampleConsortium:
+            Organizations:
+                - *Org1
+                - *Org2
+```
+
+The system channel defines the nodes of the ordering service and the set of organizations that are ordering service administrators. The system channel also includes a set of peer organizations that belong to the blockchain [consortium](../glossary.html#consortium). The channel MSP of each member of the consortium is included in the system channel. As a result, the peer organizations that belong to the consortium have the ability to create new channels and can add other organizations from the consortium as channel members.
+
+The `TwoOrgsOrdererGenesis:` profile creates a consortium named `SampleConsortium` that contains the two peer organizations in the `configtx.yaml` file, Org1 and Org2. The `Orderer` section of the profile uses the single node raft ordering service defined in the **Orderer:** section of the file. The OrdererOrg from the **Organizations:** section is made the only administrator of the ordering service. Because our only ordering node is running Fabric 2.x, we can set the orderer system channel capability to `V2_0`. The system channel uses default policies from the **Channel** section and enables `V2_0` as the channel capability level.
+
+The `TwoOrgsChannel:` profile is used to create application channels.
+```yaml
+TwoOrgsChannel:
+    Consortium: SampleConsortium
+    <<: *ChannelDefaults
+    Application:
+        <<: *ApplicationDefaults
+        Organizations:
+            - *Org1
+            - *Org2
+        Capabilities:
+            <<: *ApplicationCapabilities
+```
+
+The system channel is used by the ordering service to create application channels. The nodes of the ordering service that are defined in the system channel become the default consenter set of new channels. The channel MSPs for each organization is transferred to the new channel from the system channel. After the channel is created, ordering nodes can be added or removed from the the channel by updating the channel configuration. The same process can also be used to [add a channel member](../channel_update_tutorial.html) that does not belong to the consortium.
```

Comment:
The word `template` comes to mind. The System channel serves as a template for all application channels. Not sure if that is an appropriate metaphor in this case or not, but it makes it clear to me the difference between the system channel and application channel.

Reply:
I like it (and I think I debated using it)

The system channel is used by the ordering service as a template to create application channels.